src/intrprtr.c

```diff
@@ -234,7 +234,7 @@ static inline void FinishAndCallFakeFuncExpr(void)
     assert(STATE(IntrCoding) == 0);
 
     // code a function expression (with one statement in the body)
-    CodeFuncExprEnd(1);
+    CodeFuncExprEnd(1,0);
```

Comment:
```suggestion
    CodeFuncExprEnd(1, 0);
```

Reply:
Based on the name `useStack`, I'd actually expect this:
```suggestion
    CodeFuncExprEnd(1,1);
```